app/models/TraderProfile.scala

```diff
@@ -80,4 +80,12 @@ object TraderProfile {
         }
       case Left(errors) => Left(errors)
     }
+
+  def validateHasNirms(
+    answers: UserAnswers
+  ): EitherNec[ValidationError, Boolean] =
+    (answers.getPageValue(RemoveNirmsPage), answers.getPageValue(HasNirmsUpdatePage)) match {
```

Comment:
Does this work in case where I
1. Select No to Nirms Question
2. Select Yes to Remove Nirms
3. Go to Cya
4. On Cya, click change
5. Select Yes to Nirms Question
6. Go Back to Cya

Reply:
5. Select Yes to Nirms Question then next page should be NIRMS number right?

Reply:
Sorry, Scenario 3 is now fixed